api/hearings/data/hearingRequests.mock.data.ts

```diff
@@ -76,7 +76,6 @@ export const HEARING_REQUEST_RESULTS: HearingRequestMainModel[] = [
         partyID: 'P1',
         partyName: 'Jane and Smith',
         partyType: PartyType.IND,
-        partyChannel: 'inPerson',
```

Comment:
in this mock data, after the partyChannel: 'inPerson', is removed here, could this be added to the the individualDetails.preferredChannel.